config/routes.rb

```diff
@@ -143,6 +143,7 @@
           member { get :confirm_delete }
         end
         upload_step :reason_why
+        show_step :check_answers
```

Comment:
I think this is an edit_step as it will eventually be a form. I know you've not got the form now, but I think best to save yourself having to rename `show.html.erb` to `edit.html.erb`and update controller method names  in your next PR, as that increases the likelihood of merge conflicts.

Reply:
👍 - crossed in the ether